### PR TITLE
refactor (#184036117) refactor basicbutton to remove web …

### PR DESCRIPTION
**Pull request by nullcitizen**
…component dependency

copies css from web-components into react project. copies dom tree from web components into render.
removes component import

[Ticket TICKET_ID](https://www.pivotaltracker.com/story/show/184036117)

### CODE REVIEW

File: packages/atlas-react/src/components/BasicButton/BasicButton.scss [hunk 1]
```diff
@@ -1,7 +1,159 @@
...
+
+.BasicButton.BasicButton--align-right {
+  --#{$namespace}-button-align: flex-end;
+}
+
+.BasicButton.BasicButton--align-center {
+  --#{$namespace}-button-align: center;
+}
+
+.BasicButton {
```
Comment (nullcitizen) [inline packages/atlas-react/src/components/BasicButton/BasicButton.scss:96]: there's some cleanup that could be done in this file but I'm really hoping for the chance to revert this in January so I'm not worried about it right now

File: packages/atlas-react/src/components/BasicButton/BasicButton.scss [hunk 2]
```diff
@@ -1,7 +1,159 @@
...
+  &:disabled {
+    opacity: 0.5;
+  }
+
+  &:hover {
+    background-color: var(--#{$namespace}-button-background-color-hover);
+  }
+
+  .trailingIcon {
+    i {
```
Comment (demogar) [inline packages/atlas-react/src/components/BasicButton/BasicButton.scss]: what happens if we just use `←`, for example?

File: packages/atlas-react/src/components/BasicButton/BasicButton.scss (hunk 1 above)
Comment (demogar) [reply packages/atlas-react/src/components/BasicButton/BasicButton.scss:96]: sounds good.

**Review by demogar — APPROVED** [review]

File: packages/atlas-react/src/components/BasicButton/BasicButton.scss (hunk 2 above)
Comment (nullcitizen) [reply packages/atlas-react/src/components/BasicButton/BasicButton.scss]: ah, perhaps I should have just converted this to .trailingIcon > *

**Pull request merged**